Limelight/ViewControllers/StreamFrameViewController.m

```diff
@@ -221,12 +295,12 @@ - (void)edgeSwiped {
 - (void) connectionStarted {
     Log(LOG_I, @"Connection started");
     dispatch_async(dispatch_get_main_queue(), ^{
-        // Leave the spinner spinning until it's obscured by
-        // the first frame of video.
```

Comment:
Why did this behavior have to be removed?

Reply:
The spinner just stayed on the screen and was not obscured if the video frame was on the external screen